infrastructure/helm/quantumserverless/charts/gateway/templates/deployment.yaml

```diff
@@ -33,19 +33,19 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          command: [ "/usr/src/app/entrypoint.sh", "gunicorn", "main.wsgi:application", "--bind", "0.0.0.0:8000", "--workers=3" ]
+          args: [ "gunicorn", "main.wsgi:application", "--bind", "0.0.0.0:{{ .Values.service.port }}", "--workers=4" ]
           ports:
             - name: http
               containerPort: {{ .Values.service.port }}
               protocol: TCP
-          livenessProbe:
-            httpGet:
-              path: /metrics
-              port: http
-          readinessProbe:
-            httpGet:
-              path: /metrics
-              port: http
+#          livenessProbe:
```

Comment:
My only comment is this. Maybe we should uncomment this? :) 
or `/metrics` is not available? or do `/api/v1/`

Reply:
You are totally right. My two cents here from previous experiences is that the security team will ask us for **two different** end-points for `liveness` and `readiness` for a simple HTTP 200 and a DB access. For the DB access I think `/api/v1` can work (I would need to confirm it) but for the HTTP 200 we will need a specific end-point. That's why I commented them. If you are agree @IceKhan13 I can create an issue for this.